zeppelin-distribution/pom.xml

```diff
@@ -113,6 +113,32 @@
   </build>
 
   <profiles>
+    <profile>
+      <id>oidc</id>
+
+      <activation>
+        <activeByDefault>false</activeByDefault>
+      </activation>
+
+      <dependencies>
+        <dependency>
+            <groupId>io.buji</groupId>
+            <artifactId>buji-pac4j</artifactId>
+            <version>4.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.pac4j</groupId>
+            <artifactId>pac4j-oidc</artifactId>
+            <version>3.0.2</version>
```

Comment:
Can this version be bumped to 3.4.0 Because this pac4j-oidc client will not work with the latest keycloak release. [1189](https://github.com/pac4j/pac4j/issues/1189)
The above issue is resolved in pac4j-oidc 3.4.0

Reply:
I'll try it, hope there's no conflict dependencies :)

Reply:
There's some conflict dependencies with buji-pac4j. I've asked him to update its dependencies. If he updates it, I will bump the version in this PR. If not, we can just create a new PR to update this dependency afterwards. The PR for buji is here https://github.com/bujiio/buji-pac4j/issues/82

Reply:
Okay. Just for the information i have already tested this with the latest pac4j-oidc and pac4j-core 3.4.0 version by pulling these jars and adding them to zeppelin classpath and its working. Adding my pom below for reference

Reply:
I've done it on my part as well, it works well and looking for the buji-pac4j's pom.xml, it's ok since osgi import definition include pac4j-core from 3 to 4.

Reply:
Update is done.